middleware/test/helpers.go

```diff
@@ -264,7 +264,7 @@ func Section(t *testing.T, tc Case, sec sect, rr []dns.RR) bool {
 }
 
 // Function to make sure that CNAMES do not appear after their target records
```

Comment:
It's good practice to make the comment before a method start with the method name.  For example:

`// CNAMEOrder makes sure that CNAMES do not appear after their target records`